c/common/adapters/lock_win32.c

```diff
@@ -6,6 +6,12 @@
 #include "malloc.h"
 #include "iot_logging.h"
 
+#ifdef WINCE
+
+const char* Lock_ResultAsString[] = { "OK", "ERROR" };
```

Comment:
Is this even needed? DEFINE_ENUM_STRINGS should normally do this. Is it because of MBCS vs "UNICODE"?